### ISSUE TITLE
Load additional data into a block with a DEF file

### ISSUE THREAD

**Issue by Colin-Holehouse** [The-OpenROAD-Project/OpenDB#15]
Need to have a function available from TCL to load a DEF file that will append it's data into an existing design.

Currently apply_pdn works by generating a DEF file, then merging it with the original input DEF file. This capability is available in existing EDA tools.

**Comment by abdelrahmanhosny** [comment]
I think reading DEF does not re-initialize the database object internally. So, it should work fine if we chain multiple read_def commands. 

@ahmed-agiza can you confirm? or the database does not support reading multiple DEFs into the same db object?

**Comment by Colin-Holehouse** [comment]
Yes, but at the moment, it seems like the only choices of commands I have at the TCL level are fto create chip/block, none to add to existing data.

**Comment by ahmed-agiza** [comment]
But you can invoke defin to create another chip, or  block from another file into the same database (with another block as parent if needed), can you elabore more on your test case?

**Comment by Colin-Holehouse** [comment]
I have 2 def files containing different information for the same block. I want to load both def files resulting in 1 block containing the combination of data from both defs

The enclosed tarfile contains 2 defs for the same block gcd. The end ressult needs to be one block in OpenDB called gcd containing the floorplan informatin plus the power grid.

[combine.tar.gz](https://github.com/The-OpenROAD-Project/OpenDB/files/3775646/combine.tar.gz)

**Comment by ahmed-agiza** [comment]
I see your point, it is not there out of the box but we will try to support it soon.
